### PR TITLE
Add Weibo, VK, Linkedin Icon to Drop Down Menu

### PR DESCRIPTION
**Pull request by hoangdatngo**
## Purpose
The purpose of this PR is to add Weibo, VK, and Linkedin icon to drop down menu.
## Preview
wsgen

Preview link: https://hoangdatngo.github.io/fossasia.org/

@mariobehling, @kushthedude, @hpdang, @prateekiiest Can any mentor please review my PR?

### CODE REVIEW

**Comment by hoangdatngo** [comment]
@prateekiiest, @liveHarshit, @mariobehling, @cweitat Can any mentor please review it. I have been waiting for over 36 hours

**Comment by hoangdatngo** [comment]
@kushthedude Can you please review my PR?

**Review by cweitat — CHANGES_REQUESTED** [review]
please follow the current way of typing is . `<a href="" target=""/>` not `<a target="" href="">`

**Comment by hoangdatngo** [comment]
@mariobehling Can you please review and merge my MR?

**Review by cweitat — APPROVED** [review]

**Comment by hoangdatngo** [comment]
@liveHarshit, @cweitat Can any mentor merge my PR please

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please rebase properly, this contain possible changes from another pr not relating to the issue

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-fossasia.org
\ No newline at end of file
+fossasia.org
```
Comment (prateekj117) [inline CNAME:1]: No changes required in this file.

**Review by prateekj117 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**